pkg/container-utils/runtime-client/interface.go

```diff
@@ -61,84 +93,14 @@ type ContainerRuntimeClient interface {
 	// provided ID.
 	GetContainer(containerID string) (*ContainerData, error)
 
+	// GetContainerExtended returns the extended information of the container 
+	// identified by the provided ID.
+	GetContainerExtended(containerID string) (*ContainerExtendedData, error)
```

Comment:
I believe that in the future, `GetContainer` will disappear, and we will only use `GetContainerExtended`. Let's see, it is a discussion we can have later on.

Reply:
In this case, I suggest we change:
- `ContainerData` => `ContainerListData`
- `ContainerExtendedData` => `ContainerData` (which extends `ContainerListData`)
- `GetContainerExtended` => `GetContainer` 
Then the `GetContainer` function, which returns `ContainerData`, always returns the extended information.

Reply:
I think we should consider this now. IMO it's better to have a single `GetContainer()` container function and structure. In the end, what defines if a field goes to `ContainerData` or `ContainerExtendedData`?

Reply:
What do you think about the latest changes?